connector_importer/utils/import_utils.py

```diff
@@ -115,6 +115,10 @@ def read_lines(self):
             lines = [
                 header,
             ] + lines[int(_from or 0) : int(_to or len(lines) + 1)]
+        if len(self.delimiter) > 1:
```

Comment:
hmm 2 comments:
1. are you sure this is not a problem w/ the file? 
2. if we keep the change, please move it after line 102 in the init